.github/workflows/web.yml

```diff
@@ -0,0 +1,67 @@
+name: Web
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'binding/web/**'
```

Comment:
Is there a way to factorize these paths?

Reply:
it seems that there is no way to do so
https://github.community/t/how-to-factor-paths-in-common-for-push-and-pull-request/115967